RAW/RawModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RawModel.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+struct RawModel: Codable {
+    
+    let userid : Int?
+    let id : Int?
+    let title : String?
+    let body : String?
+    
+    enum CodingKeys: String, CodingKey {
+        case userid = "userid"
+        case id = "id"
+        case title = "title"
+        case body = "body"
+    }
+    
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
why are you overriding the init? When the generated init will do the decoding and assigning for you